app/src/main/java/com/capstone/weatherapp/WeatherDatabase.kt

```diff
@@ -0,0 +1,23 @@
+package com.capstone.weatherapp
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+@Database(entities = [CityCache::class, SingleCityCache::class], version = 1)
+abstract class WeatherDatabase : RoomDatabase() {
+    abstract fun cityDao() : CityDao
+    abstract fun singleCityDao() : SingleCityDao
+
+    companion object {
```

Comment:
This is close, your companion object is missing a few things to properly instantiate the DB